packages/flutter_secure_storage/tizen/src/secure_storage.cc

```diff
@@ -0,0 +1,196 @@
+  std::vector<std::string> keys = GetAliasList(AliasType::kData);
+  for (std::string key : keys) {
+    Delete(key);
+  }
+}
+
+bool SecureStorage::ContainsKey(const std::string &key) {
+  std::vector<std::string> keys = GetAliasList(AliasType::kData);
+  return std::find(keys.begin(), keys.end(), key) != keys.end();
+}
+
+void SecureStorage::CreateAesKeyOnce() {
+  std::vector<std::string> names = GetAliasList(AliasType::kKey);
+  for (const auto &name : names) {
+    if (name == kSecureStorageAesKey) {
+      return;
+    }
+  }
+  ckmc_policy_s policy = {
+      .password = nullptr,
+      .extractable = false,
+  };
+  ckmc_create_key_aes(256, kSecureStorageAesKey, policy);
+}
+
+std::string SecureStorage::Encrypt(const std::string &value) {
+  ckmc_raw_buffer_s plaintext;
+  plaintext.data = const_cast<unsigned char *>(
+      reinterpret_cast<const unsigned char *>(value.c_str()));
+  plaintext.size = value.length();
+
+  ckmc_param_list_h params = nullptr;
+  ckmc_generate_new_params(CKMC_ALGO_AES_GCM, &params);
+
+  std::vector<unsigned char> iv = GenerateRandomVector();
+
+  ckmc_raw_buffer_s iv_buffer;
+  iv_buffer.data = iv.data();
+  iv_buffer.size = iv.size();
+  ckmc_param_list_set_buffer(params, CKMC_PARAM_ED_IV, &iv_buffer);
+
+  ckmc_raw_buffer_s *encrypted = nullptr;
+  ckmc_encrypt_data(params, kSecureStorageAesKey, nullptr, plaintext,
+                    &encrypted);
+
+  std::string encrypted_value(encrypted->data,
+                              encrypted->data + encrypted->size);
+
+  ckmc_param_list_free(params);
+  ckmc_buffer_free(encrypted);
+
+  std::string iv_str(iv.data(), iv.data() + iv.size());
+  std::string combind = iv_str + encrypted_value;
+
+  return combind;
+}
+
+std::string SecureStorage::Decrypt(const std::string &value) {
+  std::string encrypted_value = value.substr(kInitializationVectorSize);
+  ckmc_raw_buffer_s encrypted;
+  encrypted.data = const_cast<unsigned char *>(
+      reinterpret_cast<const unsigned char *>(encrypted_value.c_str()));
+  encrypted.size = encrypted_value.length();
+
+  ckmc_param_list_h params = nullptr;
+  ckmc_generate_new_params(CKMC_ALGO_AES_GCM, &params);
+
+  std::string iv = value.substr(0, kInitializationVectorSize);
+
+  ckmc_raw_buffer_s iv_buffer;
+  iv_buffer.data = const_cast<unsigned char *>(
+      reinterpret_cast<const unsigned char *>(iv.c_str()));
+  iv_buffer.size = iv.size();
+  ckmc_param_list_set_buffer(params, CKMC_PARAM_ED_IV, &iv_buffer);
+
+  ckmc_raw_buffer_s *decrypted = nullptr;
+  ckmc_decrypt_data(params, kSecureStorageAesKey, nullptr, encrypted,
+                    &decrypted);
+
+  std::string decrypted_value(decrypted->data,
+                              decrypted->data + decrypted->size);
+
+  ckmc_param_list_free(params);
+  ckmc_buffer_free(decrypted);
+
+  return decrypted_value;
+}
+
+std::vector<unsigned char> SecureStorage::GenerateRandomVector() {
+  static std::mt19937 mt{std::random_device{}()};
```

Comment:
nit: We usually prefer this style:
```suggestion
  static std::mt19937 mt(std::random_device{}());
```